Models/Courses.py

```diff
@@ -0,0 +1,16 @@
+from sqlalchemy.ext.declarative import declarative_base
+from  sqlalchemy import Column, Integer, String
+
+
+Base = declarative_base()
+
+
+class course(Base):
+    __tablename__ = 'courses'
+    department_ids = Column(String)
```

Comment:
Why is this named `department_ids` (plural)? Should this be an array of strings?